test/error-cat.js

```diff
@@ -118,17 +118,19 @@ describe('ErrorCat', function() {
 
     beforeEach(function (done) {
       sinon.stub(error, 'log');
+      sinon.stub(error, 'report');
       sinon.stub(Boom, 'create');
       done();
     });
 
     afterEach(function (done) {
       error.log.restore();
+      error.report.restore();
       Boom.create.restore();
       done();
     });
 
-    it('should create a new boom error', function(done) {
+    it('should create a new boom error, not send to rollbar', function(done) {
```

Comment:
I believe the name change of the test is meant to reflect the removed call to `report`
https://github.com/Runnable/error-cat/pull/3/files#diff-168726dbe96b3ce427e7fedce31bb0bcL81